src/notation/view/widgets/pagesettings.cpp

```diff
@@ -159,21 +162,21 @@ void PageSettings::updateValues()
 
     blockSignals(true);
 
-    const char* suffix;
+    muse::TranslatableString suffix;
```

Comment:
In this case, you can just use `QString` here and `qtrc("global", "mm");` etc. below, and remove `.qTranslated`.

Reply:
done